### DISCUSSION TITLE
starting github desktop problem

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142137] [New to GitHub]
when I try to sign in the github desktop "Failed Retrieving Authenticated User" message is come.
how to solve this problem?

**Comment by ghostinhershell** [comment]
Hi @Mubarek44, thanks so much for reaching out in our community 👋🏾 

We have some troubleshooting recommendations in our Docs here: https://docs.github.com/en/desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop#troubleshooting-authentication-issues. The first recommendation is to try signing out and signing back in to your account on GitHub Desktop.

Take a look, try it out and let me know if that resolves your issue here. Thanks again!
